src/unittest_json.nim

```diff
@@ -0,0 +1,122 @@
+##
+## An extension of the ``unittest.nim`` library to output json for Exercism v3
+##
+
+import json, unittest, streams, strformat, strutils
+
+type
+  JsonOutputFormatter* = ref object of OutputFormatter
+    stream*: Stream
+    testErrors*: seq[string]
+    testStackTrace*: string
+    result*: ResultJson
+    ranFirstTest*: bool
+
+  JsonTestStatus* = enum
+    PASS,
+    FAIL,
+    ERROR
```

Comment:
```suggestion
    PASS = "pass",
    FAIL = "fail",
    ERROR = "error"
```
This changes the output values in the JSON file to be lowercase, as specified here https://github.com/exercism/automated-tests/blob/master/docs/interface.md#statuses